Gems/PhysX/Code/Source/System/PhysXSystem.cpp

```diff
@@ -64,8 +64,8 @@ namespace PhysX
         m_onMaterialLibraryReloadedCallback(asset);
     }
 
-    PhysXSystem::PhysXSystem(PhysXSettingsRegistryManager* registryManager, const physx::PxCookingParams& cookingParams)
-        : m_registryManager(*registryManager)
+    PhysXSystem::PhysXSystem(AZStd::unique_ptr<PhysXSettingsRegistryManager> registryManager, const physx::PxCookingParams& cookingParams)
+        : m_registryManager(AZStd::move(registryManager))
```

Comment:
@SergeyAMZN @greerdv @moraaar - worth sanity checking this, looks good to me but would be good to get confirmation

Reply:
Looks good to me. 
Interestingly I do remember seeing this memory leak before! 
Either another PR that never made development or just a leaks report somewhere..